hazelcast-jet-distribution/src/bin-filemode-755/jet-start

```diff
@@ -2,14 +2,30 @@
 
 SCRIPT_DIR=$(dirname "$0")
 #shellcheck source=bin-regular/common.sh
+
+if [[ "$1" = "--help" ]] || [[ "$1" = "-h" ]]; then
+    echo "Usage: jet-start [-d]"
+    echo "  -d, --daemon   Starts Hazelcast Jet in daemon mode"
+    exit 0
+elif [[ "$1" = "-d" ]] || [[ "$1" = "--daemon" ]]; then
+    DAEMON="true"
+fi
+
 . "$SCRIPT_DIR"/common.sh
 
 echo "########################################"
 echo "# JAVA=$JAVA"
 echo "# JAVA_OPTS=${JAVA_OPTS_ARRAY[*]}"
 echo "# CLASSPATH=$CLASSPATH"
 echo "########################################"
-echo "Starting Hazelcast Jet"
 
-set -x
-$JAVA "${JAVA_OPTS_ARRAY[@]}" -cp "$CLASSPATH" com.hazelcast.jet.server.JetMemberStarter
+if [[ "$DAEMON" = "true" ]]; then
+    mkdir -p $JET_HOME/logs
+    JET_LOG="$JET_HOME/logs/hazelcast-jet.out"
```

Comment:
we create `hazelcast-jet.out` if daemon true and `hazelcast-jet.log` otherwise

Reply:
hazelcast-jet.log is always created (by log4j logger) and it's appended. the .out file the sysout, and deleted when you restart the node.

Reply:
ah so it was intentional